playbook.yaml

```diff
@@ -0,0 +1,65 @@
+- name: Install To-Do App on new web server
+  hosts: nodes
+  remote_user: ec2-user
+  environment:
+    POETRY_HOME: /bin/poetry
+  vars_files:
+    - vault.yaml
```

Comment:
Nice work on this extension - we could arguably check this vault file into the code base now (assuming it's encrypted with a reliable password!) but given that we won't actually be using Ansible moving forward it's not going to give us much extra to do so